packages/api-gateway/src/drivers/http/adapters/admin_panel/handlers.js

```diff
@@ -1,11 +1,27 @@
 async function sayHello(req, reply) {
   const result = await this.adminPanelService.sayHello();
-
   return reply.code(200)
     .header('Content-Type', 'application/json; chartset:utf-8')
     .send({ result });
 }
-
+async function changeUserStatus(req, reply) {
+  // TODO: this is not the place for try/catch
+  try {
+    const result = await this.adminPanelService.changeUserStatus(
+      req.params.user_id,
+      req.body.status,
+      req.body.reason,
+    );
+    return reply.code(200)
+      .header('Content-Type', 'application/json; chartset:utf-8')
+      .send({result});
```

Comment:
Eslint is throwing errors in this line because it wants spaces before and after the curly braces.
```suggestion
      .send({ result });
```

Reply:
mm that´s weird, I just ran yarn:lint and the result is 
`yarn run v1.22.18
$ eslint .
Done in 2.47s.`